package.json

```diff
@@ -16,6 +16,7 @@
   },
   "dependencies": {
     "@emotion/eslint-plugin": "^11.10.0",
+    "@tidio/eslint-plugin-tidio-additional-rules": "^2.0.0",
```

Comment:
Q: skoro mamy `@tidio`, to czy nie lepiej `eslint-plugin-additional-rules`?

mielibysmy `@tidio/additional-rules/imports-order-for-less-files` zamiast `@tidio/tidio-additional-rules/imports-order-for-less-files`